### ISSUE TITLE
System Notification for:  DigiCert (Intent to Decommission Multiple CAs)

### ISSUE THREAD

**Issue by ryancdickson** [GSA/fpki-guides#446]
change_type: Intent to Decommission Multiple CAs
notice_date: March 25, 2019
start_datetime: 
end_datetime: 
system: Symantec Class 3 SSP Intermediate CA - G3 (managed by DigiCert)
change_description: DigiCert is planning to decommission several CAs chaining to Symantec Class 3 SSP Intermediate CA - G3. Any remaining active certificates issued from the CAs listed below will be revoked before CA decommissioning.
contact: steve dot medin at digicert dot com
ca_certificate_hash: Multiple CA certificates will be revoked before decommissioning. Comma-separated list of CA hashes planned for revocation/decommissioning: 2938CC1B586C8B1A76832973389AB25C53F3658F, 3DFB1CB09BE5D1430DF9CE8E04F501ADB86CA176, 1EC98449B18472D6C2F516A4D0976350D5DD829A, CF9229CB50BF5DC25C156C4F825A67E2964236C8, FAED5B3AA85BFEA0BA8BA884689706044DFC0EC9, D4C786468511920E538C24384B3132D07DBACAF0, D514632CE34BF94C816A565DBB8FBB6BAD5FF335, 82E6536172F69E30786C9670840DB37077B5C08C, and F8912B3C9F3EB30F8A8D5A14E8038BD9BC6B771D
ca_certificate_issuer: Symantec Class 3 SSP Intermediate CA - G3
ca_certificate_subject: Multiple CA certificates will be revoked before decommissioning. Comma-separated list of CA Subject Names planned for revocation/decommissioning: CSC CA - 2, CSRA FBCA C4 Device CA, CSRA FBCA C4 CA, CSRA FBCA C3 Device CA, CSRA FBCA C3 CA, SureID Inc. CA2, SureID Inc. Device CA1, Eid Passport LRA Content Signer CA 3, and Eid Passport LRA Device 2 CA
cdp_uri: http://ssp-crl.symauth.com/STNSSP/Class3SSPCAG3.crl
aia_uri: http:/​/​ssp-aia.​symauth.​com/​STNSSP/​Certs_​issued_​to_​Class3SSPCA-G3.​p7c
sia_uri: 
ocsp_uri: http://ssp-ocsp.symauth.com

**Comment by ryancdickson** [comment]
Update: Revocation still pending. Leaving issue open.

**Comment by ryancdickson** [comment]
Revocation took place on July 3, 2019 for all CAs except CSC CA - 2 (expired on  June 19, 2019) and CSRA FBCA C3 CA (still active).

**Comment by ryancdickson** [comment]
CSRA FBCA C3 CA was revoked on 7/10. Will close issue once notification is merged into staging.
